From dca3fa0dab2ad688e19c5881fa0094dc4bbbce23 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:16 -0400
Subject: [redhat] kabi: Adding symbol nla_memcpy (lib/nlattr.c)

Message-id: <c15c37686b6f9a892fb770582bda3364c076f97f.1410510773.git.poros@redhat.com>
Patchwork-id: 95996
O-Subject: [PATCH RHEL7.1 136/233] kabi: Adding symbol nla_memcpy (lib/nlattr.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Netlink API

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/nla_memcpy
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/nla_memcpy
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/nla_memcpy

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/nla_memcpy  b/redhat/kabi/kabi-module/kabi_ppc64/nla_memcpy 
new file mode 100644
index 0000000..82ff77f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/nla_memcpy  
@@ -0,0 +1,2 @@
+#1-
+0xf1db1704 nla_memcpy vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/nla_memcpy  b/redhat/kabi/kabi-module/kabi_s390x/nla_memcpy 
new file mode 100644
index 0000000..82ff77f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/nla_memcpy  
@@ -0,0 +1,2 @@
+#1-
+0xf1db1704 nla_memcpy vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/nla_memcpy  b/redhat/kabi/kabi-module/kabi_x86_64/nla_memcpy 
new file mode 100644
index 0000000..82ff77f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/nla_memcpy  
@@ -0,0 +1,2 @@
+#1-
+0xf1db1704 nla_memcpy vmlinux EXPORT_SYMBOL
-- 
1.7.1